app/controllers/api/v1/items/search_controller.rb

```diff
@@ -3,11 +3,26 @@ def show
     attribute = params.keys.first
     keyword = params[attribute]
     if attribute == "name" || attribute == "description"
-      item = Item.where("items.#{attribute} ILIKE ?", "%#{keyword}%").limit(1)
+      item = Item.find_by("items.#{attribute} ILIKE ?", "%#{keyword}%")
     elsif attribute == "created_at" || attribute == "updated_at"
-      item = Item.where("DATE(#{attribute}) LIKE ?", "%#{keyword}%").limit(1)
+      keyword = keyword.to_s
+      item = Item.find_by("DATE(#{attribute}) = ?", "#{keyword}")
     else
-      item = Item.where("#{attribute} = ?", "#{keyword}").limit(1)
+      item = Item.find_by("#{attribute} = ?", "#{keyword}")
+    end
+    render json: ItemSerializer.new(item)
+  end
+
+  def index
+    attribute = params.keys.first
+    keyword = params[attribute]
+    if attribute == "name" || attribute == "description"
+      item = Item.where("items.#{attribute} ILIKE ?", "%#{keyword}%")
+    elsif attribute == "created_at" || attribute == "updated_at"
+      keyword = keyword.to_s
+      item = Item.where("DATE(#{attribute}) = ?", "#{keyword}")
+    else
+      item = Item.where("#{attribute} = ?", "#{keyword}")
```

Comment:
Where is find for returning multiple results because that is how the FE is expecting the data to arrive!